app/src/main/java/io/github/wulkanowy/ui/modules/timetable/TimetableAdapter.kt

```diff
@@ -26,48 +27,78 @@ import kotlin.concurrent.timer
 
 class TimetableAdapter @Inject constructor() : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
 
-    private enum class ViewType(val id: Int) {
-        ITEM_NORMAL(1),
-        ITEM_SMALL(2)
+    private enum class ViewType {
+        ITEM_NORMAL,
+        ITEM_SMALL
     }
 
-    var items = mutableListOf<Timetable>()
-        set(value) {
-            field = value
-            resetTimers()
-        }
-
     var onClickListener: (Timetable) -> Unit = {}
 
-    var showWholeClassPlan: String = "no"
+    private var showWholeClassPlan: String = "no"
+
+    private var showGroupsInPlan: Boolean = false
+
+    private var showTimers: Boolean = false
+
+    private val timers = mutableMapOf<Int, Timer?>()
+
+    private val items = mutableListOf<Timetable>()
 
-    var showGroupsInPlan: Boolean = false
+    fun submitList(
+        newTimetable: List<Timetable>,
+        showWholeClassPlan: String = "no",
+        showGroupsInPlan: Boolean = false,
+        showTimers: Boolean = false
```

Comment:
```suggestion
        showWholeClassPlan: String = this.showWholeClassPlan,
        showGroupsInPlan: Boolean = this.showGroupsInPlan,
        showTimers: Boolean = this.showTimers
```

dzięki temu wartości domyślne będą tylko w jednym miejscu zamiast w dwóch